lib/vendor/lime/lime.php

```diff
@@ -155,6 +161,31 @@ public function __destruct()
     }
 
     flush();
+
+    self::$allExitCode |= $this->getExitCode();
```

Comment:
this is a binary operation, done on an integer value, we should cast it to a bool before, or rename `getExitCode()` to `isTestFailed()`

WDYT?